### PR TITLE
Merge PS 5.7.31, Part 1 

### PR DESCRIPTION
**Pull request by dutow**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1,14 +1,6 @@
 MYSQL_VERSION_MAJOR=5
 MYSQL_VERSION_MINOR=7
 <<<<<<< HEAD
-MYSQL_VERSION_PATCH=30
-MYSQL_VERSION_EXTRA=-33
-PXC_VERSION_EXTRA=43
```
Comment (kamil-holubicki) [inline VERSION:5]: What does PXC_VERSION_EXTRA mean? It was 57 originally (indicating 5.7 ver of server?) then changed to 43 (what for?)

File: build-ps/ubuntu/control [hunk 2]
```diff
@@ -189,16 +188,6 @@ Depends: percona-xtradb-cluster-client-5.7 (>= ${binary:Version}),
...
          iproute
-||||||| merged common ancestors
-Depends: percona-server-client-5.7 (= ${binary:Version}),
-         ${shlibs:Depends}, ${misc:Depends},
-         psmisc
-=======
-Depends: percona-server-client-5.7 (= ${binary:Version}),
-         ${shlibs:Depends}, ${misc:Depends},
-         psmisc,
-         openssl
```
Comment (kamil-holubicki) [inline build-ps/ubuntu/control]: openssl missing

File: packaging/rpm-docker/mysql.spec.in [hunk 3]
```diff
@@ -1,10 +1,4 @@
-<<<<<<< HEAD
 # Copyright (c) 2000, 2018, Oracle and/or its affiliates. All rights reserved.
-||||||| merged common ancestors
-# Copyright (c) 2000, 2019, Oracle and/or its affiliates. All rights reserved.
-=======
-# Copyright (c) 2000, 2020, Oracle and/or its affiliates.
```
Comment (kamil-holubicki) [inline packaging/rpm-docker/mysql.spec.in]: Should be resolved as 
Copyright (c) 2000, 2020, Oracle and/or its affiliates.

File: packaging/rpm-fedora/mysql.spec.in [hunk 4]
```diff
@@ -1,10 +1,4 @@
-<<<<<<< HEAD
 # Copyright (c) 2000, 2016, Oracle and/or its affiliates. All rights reserved.
-||||||| merged common ancestors
-# Copyright (c) 2000, 2019, Oracle and/or its affiliates. All rights reserved.
-=======
-# Copyright (c) 2000, 2020, Oracle and/or its affiliates.
```
Comment (kamil-holubicki) [inline packaging/rpm-fedora/mysql.spec.in]: Should be resolved as
Copyright (c) 2000, 2020, Oracle and/or its affiliates.

File: packaging/rpm-sles/mysql.spec.in [hunk 5]
```diff
@@ -1,10 +1,4 @@
-<<<<<<< HEAD
 # Copyright (c) 2000, 2018, Oracle and/or its affiliates. All rights reserved.
-||||||| merged common ancestors
-# Copyright (c) 2000, 2019, Oracle and/or its affiliates. All rights reserved.
-=======
-# Copyright (c) 2000, 2020, Oracle and/or its affiliates.
```
Comment (kamil-holubicki) [inline packaging/rpm-sles/mysql.spec.in]: should be resolved as
Copyright (c) 2000, 2020, Oracle and/or its affiliates.

File: packaging/rpm-oel/mysql.spec.in [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2000, 2020, Oracle and/or its affiliates.
+# Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (kamil-holubicki) [inline packaging/rpm-oel/mysql.spec.in]: This should be
Copyright (c) 2000, 2020, Oracle and/or its affiliates.

**Review by kamil-holubicki — COMMENTED** [review]
According to this 
https://confluence.percona.com/pages/viewpage.action?spaceKey=PXC&title=Upstream+merges
We should leave man a doc as it is when doing the merge (merge with conflicts). Doing so will give cleaner/easier to understand conflict resolution commit.

File: VERSION (hunk 1 above)
Comment (dutow) [reply VERSION:5]: It relaced WSREP_INTERFACE_VERSION, and based on the use, it should be the last part of the version string, which is currently 43.

**Review by kamil-holubicki — APPROVED** [review]
LGTM

File: build-ps/debian/control [hunk 7]
```diff
@@ -189,16 +188,6 @@ Depends: percona-xtradb-cluster-client-5.7 (>= ${binary:Version}),
...
          iproute
-||||||| merged common ancestors
-Depends: percona-server-client-5.7 (= ${binary:Version}),
-         ${shlibs:Depends}, ${misc:Depends},
-         psmisc
-=======
-Depends: percona-server-client-5.7 (= ${binary:Version}),
-         ${shlibs:Depends}, ${misc:Depends},
-         psmisc,
-         openssl
```
Comment (kamil-holubicki) [inline build-ps/debian/control]: oops, I think we missed this :(

**Pull request merged**